paddle/fluid/platform/device_context.cc

```diff
@@ -502,6 +504,7 @@ MKLDNNDeviceContext::MKLDNNDeviceContext(CPUPlace place)
     : CPUDeviceContext(place), p_blobmap_() {
   p_blobmap_.reset(new BlobMap());
   p_mutex_.reset(new std::mutex());
+  if (FLAGS_onednn_use_input_mem_format) use_input_mem_format_ = true;
```

Comment:
why not use: __use_input_mem_format = FLAGS_onednn_use_input_mem_format;__  ?

Reply:
Good  idea.